src/resource_dasm.cc

```diff
@@ -2711,6 +2740,25 @@ int main(int argc, char* argv[]) {
       } else if (!strncmp(argv[x], "--filename-format=", 18)) {
         exporter.filename_format = &argv[x][18];
       
+      } else if (!strncmp(argv[x], "--icon-family-format=", 21)) {
+        auto formats = split(&argv[x][21], ',');
+        if (formats.size() == 0) {
+          throw logic_error("--icon-family-format needs a value");

```

Comment:
There isn't a clearly-established pattern for what we do for invalid CLI arguments (I'll fix this), but I think we should throw invalid_argument here (and below).

Reply:
Fixed (including `--decode-single-resource` where I copied the `logic_error` from).